website/docs/troubleshooting/emails.md

```diff
@@ -25,3 +25,26 @@ To run this command with Kubernetes, use
 ```shell
 kubectl exec -it deployment/authentik-worker -c worker -- ak test_email [...]
 ```
+
+## Google Workspace Email Relay Configuration
+
+To send email through Google SMTP servers, it is often easiest to use [Google's Relay option](https://support.google.com/a/answer/2956491). For Authentik, Google has easier to follow documentation available at [Send email from a printer, scanner, or app](https://support.google.com/a/answer/176600?hl=en). Setting this up can be tricky as there is conflicting information about correct settings online. While setting up a service account and an App Password is possible, it is not needed to successfully send emails from Authentik.
+
+First, confirm the outbound IP address that Authentik will use to send emails with. Follow Google's documentation to add the IP address or addresses to the "SMTP relay service" options in your workspace's Gmail settings.
```

Comment:
```suggestion
First, confirm the outbound IP address that authentik will use to send emails with. Follow Google's documentation to add the IP address or addresses to the "SMTP relay service" options in your workspace's Gmail settings.
```

Reply:
Can we tell them where/how to confirm that the outbound IP address is, that authentik will use?

Reply:
There's a few ways, ranging from manually inspecting things with `ip`, or `curl`'ing ifconfig.co, icanhazip.com, etc. Any preference?